projects/xpla/index.js

```diff
@@ -0,0 +1,35 @@
+const axios = require('axios');
+
+async function staking(timestamp, ethBlock, chainBlocks) {
+  const { bonded_tokens } = (
+    await axios.get(`https://dimension-lcd.xpla.io/staking/pool`)
+  ).data.result;
+  
+  const value = bonded_tokens / 1000000000000000000
+  console.log("staking:", value, bonded_tokens)
+
+  return { xpla : value }
+}
+
+async function xpla(timestamp, ethBlock, chainBlocks) {
+  const totalSupply = (
+    await axios.get(`https://dimension-lcd.xpla.dev/cosmos/bank/v1beta1/supply/axpla`)
+  ).data.amount.amount;
+
+  const currentCirculationgSupply = (
+    await axios.get(`https://dimension-fcd.xpla.dev/v1/bank/xplasupply/circulating`)
+  ).data;
+
+  const result = totalSupply / 1000000000000000000 - currentCirculationgSupply
```

Comment:
bit confused by what is happening here

Reply:
![photo_2023-05-30_10-57-26](https://github.com/DefiLlama/DefiLlama-Adapters/assets/95893422/90ae68e0-56de-41bf-bdbc-626c670ed713)

https://explorer.xpla.io/

You can see this item on the Xpla explorer page.
Can't use that code?

Reply:
@park1847 sorry, but you cant use total circulating of your token as tvl, for something to be counted as tvl, it needs to be tokens locked in smart contracts on the chain like a dex or lending protocol.

Reply:
B is the result. B can't be the result?

[C = total supply] minus [ A = current circulating supply] is [ B = non-circulating supply].